### ISSUE TITLE
I think phpunit 8.4 removed PHPUnit\Util\InvalidArgumentHelper

### ISSUE THREAD

**Issue by smknstd** [rdohms/phpunit-arraysubset-asserts#9]
```
  Error: Class 'PHPUnit\Util\InvalidArgumentHelper' not found

vendor/dms/phpunit-arraysubset-asserts/src/ArraySubsetAsserts.php:36
```

**Comment by smknstd** [comment]
https://github.com/sebastianbergmann/phpunit/blob/8.4.0/src/Util/InvalidArgumentHelper.php

**Comment by alecwcp** [comment]
`PHPUnit\Util\InvalidArgumentHelper` was marked as *internal* - and so wasn't subject to the same backwards compatibility promises.
This has been addressed in this commit https://github.com/rdohms/phpunit-arraysubset-asserts/commit/2b1fd90268ed31d8609ccc6569d9a683373249f2
That commit also makes the required version of PHPUnit to `^9.0` - so requires updating your project to PHPUnit at the same time. We're not ready to do that but want to make use of this assertion - I've made a fork and hope to get the time to address this but allow supporting PHPUnit 8 and 9, I'll post here if I make any progress with this.

**Comment by rdohms** [comment]
@alecwcp i'm looking into fixing that and launching a 0.1.1 version

**Comment by alecwcp** [comment]
Fantastic, thanks @rdohms

**Comment by rdohms** [comment]
Fixed in [v0.1.1](https://github.com/rdohms/phpunit-arraysubset-asserts/releases/tag/v0.1.1) for PHPunit 8.4 and in [v0.2.0](https://github.com/rdohms/phpunit-arraysubset-asserts/releases/tag/v0.2.0) for PHPunit v9
